ghost/admin/app/styles/layouts/members.css

```
@@ -50,1768 +43,698 @@
+/* Member details
+/* ----------------------------------------- */
+
+label[for="member-description"] + p {
+    margin: 0 0 4px;
 }
 
-.gh-list-with-helpsection {
-    height: unset;
-    margin: 0 -48px;
+.gh-member-settings .gh-main-section.columns-3 {
+    grid-column-gap: 48px;
```

Stylelint (17.9.0)

[error] Expected "grid-column-gap" to be "column-gap" (property-no-deprecated)

(property-no-deprecated)